docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -59,6 +59,8 @@ Adding the following snippet to a Quadlet file extends the systemd timeout to 15
 TimeoutStartSec=900
 ```
 
+Qualdet requires the use of cgroupv2, you can check the version with `podman info --format {{.Host.CgroupsVersion}}` on the system.
```

Comment:
Typo: "Qual"

Also: the string "cgroup[s]v2" does not appear anywhere under docs/source/markdown. I see it always with a space: "cgroup v2" and "cgroups v2" (sometimes singular, sometimes plural, it seems we should pick one but that's beyond the scope of this PR).

Also: I think ~~we're~~ it is preferred to avoid pronouns in man pages. Maybe "Use podman info ... to check"?